tests/e2e/users/add-user.wdio-spec.js

```diff
@@ -0,0 +1,105 @@
+const utils = require('../../utils');
+const faker = require('faker');
+const chai = require('chai');
+const usersAdminPage = require('../../page-objects/admin/user.wdio.page');
+const placeFactory = require('../../factories/cht/contacts/place');
+const loginPage = require('../../page-objects/login/login.wdio.page');
+const personFactory = require('../../factories/cht/contacts/person');
+
+const onlineUserRole = 'Program Officer';
+const offlineUserRole = 'CHW';
+const username = faker.name.firstName().toLocaleLowerCase();
+const password = 'Jacktest@123';
+const incorrectpassword = 'Passwor';
+const places = placeFactory.generateHierarchy();
+const district_hospital = places.find((place) => place.type === 'district_hospital');
```

Comment:
This is a good place to create a utility function. We have this kind of thing in a few other places. You could create something that works like:
```
findDistrictHospital(places)
```

The core benefit is not having that "magic string" value of `district_hospital` copied around. (A TypeScript enum might help here, but that's not worth the refactor right now).

Please also change the variable name to match JS standard camel case: `districtHospital`